src/pages/404.js

```diff
@@ -7,28 +7,75 @@ import styled from 'styled-components'
 import remcalc from 'remcalc'
 import ReactMarkdown from 'react-markdown'
 
+import BlueBackground from '../components/Common/BlueBackground'
 import { SectionTitle, BodyPrimary } from '../components/Typography'
-import Button from '../components/Common/Button'
 import Layout from '../components/layout'
 
-const HomePageLink = styled(Button)`
-  width: ${remcalc(145)};
-  display: flex;
-  justify-content: center;
+import { LogoStyleContext } from '../context/PageContext'
+
+import illustration from '../images/404-illustration.svg'
+
+// Layout
+
+const StyledBlueBackground = styled(BlueBackground)`
+  margin-top: -${remcalc(36)};
 `
 
-const StyledBodyPrimary = styled(BodyPrimary)`
-  padding-bottom: ${({ theme }) => theme.space[6]};
+const StyledRow = styled(Row)`
+  align-items: center;
 `
 
-const StyledCol = styled(Col)`
-  margin: ${({ theme }) => theme.space[5]} 0;
+const TextCol = styled(Col).attrs({
+  width: [1, 1, 1, 1, 0.5, 0.5]
+})`
+  padding-top: ${({ theme }) => theme.space[4]};
+  ${breakpoint('smallTablet')`
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[7]};
+    padding-bottom: ${({ theme }) => theme.space[8]};
+  `}
+`
 
+const IllustrationCol = styled(Col).attrs({
+  width: [1, 1, 1, 1, 0.5, 0.5]
+})`
+  ${breakpoint('smallTablet')`
+    padding-top: ${({ theme }) => theme.space[4]};
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
   ${breakpoint('tablet')`
-    margin: ${({ theme }) => theme.space[8]} 0
+    padding-top: unset;
+    padding-bottom: unset;
   `}
 `
 
+// Paragraphs / Texts
+
+const TitleHeadline = props => (
+  <SectionTitle {...props} as="h1" reverse="true" />
+)
+
+const CopyText = props => <BodyPrimary {...props} reverse="true" />
+
+const LinksTitle = styled(BodyPrimary).attrs({
+  reverse: 'true',
+  bold: 'true'
```

Comment:
Not saying this is wrong but the standard on this project is to pass in props at JSX level rather than in styled-component methods. Do you know of any differences in the outcomes?

Reply:
I don't think there are any - [styled-components docs](https://www.styled-components.com/docs/api#attrs)
It just feels like they belong here because `reverse` and `bold` are very much a styling concern.

Reply:
Do you mean that props should only be used for actions that concern behaviour etc.? 

I would agree in general although in the interest of consistency I'd prefer if we kept them defined on component level

Reply:
> Do you mean that props should only be used for actions that concern behaviour etc.?

Not at all, just that I don't want to split the styling with the `styled-components` template string part at the top and the `modifiers` at the bottom.
At least not on one element, other elements do still have styling props at the bottom, I didn't go that far.